proxies/events-graphql-proxy/src/index.ts

```diff
@@ -1,99 +1,28 @@
-import http from 'http';
-import { ApolloServer } from '@apollo/server';
-import { expressMiddleware } from '@apollo/server/express4';
-import { ApolloServerPluginDrainHttpServer } from '@apollo/server/plugin/drainHttpServer';
-import { RewriteFrames } from '@sentry/integrations';
-import * as Sentry from '@sentry/node';
-import { json } from 'body-parser';
-import cors from 'cors';
 import * as dotenv from 'dotenv';
-import express from 'express';
-import depthLimit from 'graphql-depth-limit';
-import ContextValue from './context/context-value';
-import schema from './schema';
-import apolloLoggingPlugin from './utils/apolloLoggingPlugin';
-import sentryLoggingPlugin from './utils/sentryLoggingPlugin';
 
-const OK = 'OK';
-const SERVER_IS_NOT_READY = 'SERVER_IS_NOT_READY';
-const GRAPHQL_PATH = '/proxy/graphql';
+import type { ServerConfig } from 'events-helsinki-graphql-proxy-server/src';
+import { startServer } from 'events-helsinki-graphql-proxy-server/src';
+import EventContext from './context/EventContext';
+import schema from './schema';
 
 dotenv.config();
 
-Sentry.init({
-  dsn: process.env.GRAPHQL_PROXY_SENTRY_DSN,
-  environment: process.env.GRAPHQL_PROXY_SENTRY_ENVIRONMENT,
-  integrations: [
-    // used for rewriting SourceMaps from js to ts
-    // check that sourcemaps are enabled in tsconfig.js
-    // read the docs https://docs.sentry.io/platforms/node/typescript/
-    new RewriteFrames({
-      root: process.cwd(),
-    }),
-  ],
-});
-
-const port = process.env.GRAPHQL_PROXY_PORT || 4100;
+const trueEnv = ['true', '1', 'yes'];
 
-let serverIsReady = false;
-
-const signalReady = () => {
-  serverIsReady = true;
+const config: ServerConfig = {
+  sentryDsn: process.env.GRAPHQL_PROXY_SENTRY_DSN,
+  sentryEnvironment: process.env.GRAPHQL_PROXY_SENTRY_ENVIRONMENT,
+  debug: trueEnv.includes(process.env.GRAPHQL_PROXY_DEBUG ?? 'false'),
+  serverPort: Number(process.env.GRAPHQL_PROXY_PORT) || 4100,
+  disableWinstonLogging: trueEnv.includes(
+    process.env.GRAPHQL_PROXY_DISABLE_WINSTON_LOGGING ?? 'false'
+  ),
```

Comment:
introspection should be configurable with the environment variable.

```
{"errors":[{"extensions":{"code":"GRAPHQL_VALIDATION_FAILED","http":{"headers":{},"status":400},"validationErrorCode":"INTROSPECTION_DISABLED"},"locations":[{"column":7,"line":3}],"message":"GraphQL introspection is not allowed by Apollo Server, but the query contained __schema or __type. To enable introspection, pass introspection: true to ApolloServer in production"}],"level":"error","message":"Apollo encountered errors:","operationName":"IntrospectionQuery","requestId":"df29eb4c-7dd2-47d0-b1f3-627d51307e05"}
```

Reply:
also the `introspection: isProd,` should be the opposite! 😂

Reply:
I added a new env variable for it